firmware/Makefile

```diff
@@ -44,3 +45,9 @@ endif
 
 st-flash: build
 	st-flash --reset write $(BUILD_DIR)/$(PROJECT_NAME).bin 0x08000000
+
+debug-openocd:
+	openocd -f interface/stlink.cfg -f target/stm32f$(PLATFORM_FAMILY)x.cfg
+
+debug-gdb:
+	arm-none-eabi-gdb "$(BUILD_DIR)/main.elf" -ex "target extended-remote localhost:3333" -ex "monitor reset halt"
```

Comment:
is this port guaranteed by openocd? Is there a way to specify it in the openocd command?

Reply:
The default port is 3333 but we can specify another using `-c "gdb_port XXXX"` in the openocd command and update the gdb command accordingly. Are you thinking of using another port number for this?

Reply:
Ok, just wanted to make sure 3333 is the documented default. Can you add that flag to the ocd command to make it explicit?